### PR TITLE
test: add integration test for the new `swap` customization

### PR DESCRIPTION
**Pull request by mvo5**
This commit adds an integration test for the new swap disk customization that got added to the `images` library in PR https://github.com/osbuild/images/pull/1072

### CODE REVIEW

File: test/test_manifest.py [hunk 1]
```diff
@@ -633,3 +633,97 @@ def test_manifest_disk_customization_btrfs(tmp_path, build_container):
...
+    config = {
+        "customizations": {
+            "disk": {
+                "partitions": [
+                    {
+                        "type": "lvm",
+                        "minsize": "10 GiB",
+                        "logical_volumes": [
+                            {
+                                # XXX: drop once https://github.com/osbuild/images/pull/1088 is merged
```
Comment (ondrejbudai) [inline test/test_manifest.py]: This can be dropped now.

File: test/testutil.py [hunk 2]
```diff
@@ -159,6 +159,12 @@ def maybe_create_disk_customizations(cfg, tc):
                             "minsize": "10 GiB",
                             "fs_type": "xfs",
                             "mountpoint": "/",
+                        },
+                        {
+                            # XXX: drop once https://github.com/osbuild/images/pull/1088 is merged
```
Comment (ondrejbudai) [inline test/testutil.py]: Same here.

**Review by ondrejbudai — CHANGES_REQUESTED** [review]
Looks great! I appreciate the simplification of the list of test cases, it's now much more straightforward. I found two minor issues inline.

File: test/test_manifest.py (hunk 1 above)
Comment (mvo5) [reply test/test_manifest.py]: Indeed, noticed too and fixed now :)THANKS!

**Pull request merged**